Omraj Sharma/milestone-01/q3/even-or-odd-number.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+int main()
+{
+    int x;
+
+    std::cout << "Enter a number : ";
```

Comment:
I think, using namespace std; at once is a better option. 
than prefixing std:: all the time